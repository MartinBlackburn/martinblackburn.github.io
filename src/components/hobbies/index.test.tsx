// libraries
import { render } from "@testing-library/react";

// component under test
import Hobbies from "./";

// mocks
jest.mock("./hobby", () => () => <div>hobby</div>);

describe("Hobbies component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<Hobbies />);

        expect(asFragment()).toMatchSnapshot();
    });
});
