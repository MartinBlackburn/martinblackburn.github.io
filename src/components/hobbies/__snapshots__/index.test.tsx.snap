// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hobbies component renders correctly 1`] = `
<DocumentFragment>
  <section
    class="hobbies"
  >
    <h1>
      Things I do for fun:
    </h1>
    <div
      class="hobbies__list"
    >
      <div>
        hobby
      </div>
      <div>
        hobby
      </div>
      <div>
        hobby
      </div>
      <div>
        hobby
      </div>
    </div>
  </section>
</DocumentFragment>
`;
